docs/introduction.rst

```diff
@@ -183,7 +204,7 @@ branch of the Git repository of the package:
 .. _`Verification of the installation`:
 
 Verification of the installation
-""""""""""""""""""""""""""""""""
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can verify that the pywbemtools package and its dependent packages are
 installed correctly by invoking pywbemcli. Invoking with the ``--version``
```

Comment:
Lets add that --version returns the current versions of pywbemtools and pywbem.  For example:

Reply:
That's already in the text.